source/jormungandr/jormungandr/parking_space_availability/bss/bss_provider_manager.py

```diff
@@ -43,12 +43,19 @@ def __init__(self):
 
     def handle_places(self, places):
         for place in places or []:
-            if place['embedded_type'] == 'poi' and place['poi']['poi_type']['id'] == 'poi_type:amenity:bicycle_rental':
-                provider = self.find_provider(place['poi'])
-                if provider:
-                    place['poi']['stands'] = provider.get_informations(place['poi'])
+            if 'poi_type' in place:
+                place = self.handle_poi(place)
```

Comment:
I used the return value to update the place variable. You think it modifies it by ref ?